contracts/JBFundingCycleStore.sol

```diff
@@ -97,22 +99,37 @@ contract JBFundingCycleStore is JBControllerUtility, IJBFundingCycleStore {
 
     @return _fundingCycle The queued funding cycle.
   */
-  function queuedOf(uint256 _projectId) public view override returns (JBFundingCycle memory) {
+  function queuedOf(uint256 _projectId)
+    external
+    view
+    override
+    returns (JBFundingCycle memory _fundingCycle)
+  {
     // The project must have funding cycles.
     if (latestConfigurationOf[_projectId] == 0) return _getStructFor(0, 0);
 
     // Get a reference to the configuration of the standby funding cycle.
     uint256 _fundingCycleConfiguration = _standbyOf(_projectId);
 
     // If it exists, return it's funding cycle.
-    if (_fundingCycleConfiguration > 0)
-      return _getStructFor(_projectId, _fundingCycleConfiguration);
-
-    // Get a reference to the latest stored funding cycle configuration for the project.
-    _fundingCycleConfiguration = latestConfigurationOf[_projectId];
-
-    // Resolve the funding cycle for the for the latest configured funding cycle.
-    JBFundingCycle memory _fundingCycle = _getStructFor(_projectId, _fundingCycleConfiguration);
+    if (_fundingCycleConfiguration > 0) {
+      _fundingCycle = _getStructFor(_projectId, _fundingCycleConfiguration);
+      if (_isApproved(_projectId, _fundingCycle)) return _fundingCycle;
```

Comment:
Since you override _fundingCycle on L120, it seems like it would be slightly more readable to just do:

// If the funding cycle has already been approved, return it directly.
if (_isApproved(_projectId, _fundingCycleConfiguration))  {
  return _fundingCycle;
}

Reply:
got it